### PR TITLE
Add sorting button options to tag show template

### LINKED ISSUE #3462: Add sorting button options to tag show template
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

This will add some nice buttons to tag pages such as these: https://publiclab.org/tag/mobius

Which will allow for sorting what's displayed!

screen shot 2018-09-25 at 11 58 38 am

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/views/tag/show.html.erb](https://github.com/publiclab/plots2/blob/master/app/views/tag/show.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -2,11 +2,11 @@
 
 <div class="col-md-12">
   <%= render partial: 'tag/show/header' %>
-  
+
   <% if @tags.first && params[:id][-1..-1] != "*" %>
-    <%= render :partial => "tag/graph", locals: { tags: @tags } %>
+    <%= render partial: "tag/graph", locals: { tags: @tags } %>
   <% end %>
-  
+
   <%= render partial: 'tag/show/contributors' %>
 
   <% unless @tags.try(:first).try(:parent).nil? %>
@@ -20,7 +20,10 @@
   <%= render partial: "tag/show/related_tags" %>
   <%= render partial: 'tag/show/nav_tabs' %>
   
-  <br />
+  <p style="padding-top:8px;">
+    <a class="btn btn-sm btn-default" href="?order=views"><i class="fa fa-eye"></i> by views</a>
+    <a class="btn btn-sm btn-default" href="?order=likes"><i class="fa fa-star"></i> by likes</a>
+  </p>
 
   <%= render partial: 'tag/show/tab_content' %>
 
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by msarit**
fixes #3462; Merge AFTER #3419 please

### CODE REVIEW

**Comment by ebarry** [comment]
Exciting!

**Comment by msarit** [comment]
Thanks @ebarry 👋

**Comment by jywarren** [comment]
Keeping an eye on #3430 -- once that solves #3419 we can merge this one!

File: app/views/tag/show.html.erb [hunk 1]
```diff
@@ -20,7 +21,11 @@
   <%= render partial: "tag/show/related_tags" %>
   <%= render partial: 'tag/show/nav_tabs' %>
   
-  <br />
+  <!-- <br /> -->
```
Comment (SidharthBansal) [inline app/views/tag/show.html.erb]: Please remove `` <!-- <br /> -->``

File: app/views/tag/show.html.erb [hunk 2]
```diff
@@ -4,7 +4,8 @@
   <%= render partial: 'tag/show/header' %>
   
   <% if @tags.first && params[:id][-1..-1] != "*" %>
-    <%= render :partial => "tag/graph", locals: { tags: @tags } %>
+    <%#= render :partial => "tag/graph", locals: { tags: @tags } %>
```
Comment (SidharthBansal) [inline app/views/tag/show.html.erb]: Please remove `` line 7``

**Comment by SidharthBansal** [comment]
Thanks for contributing at Public Labs.
This issue seems to be completed by someone else in https://github.com/publiclab/plots2/pull/3833. You can look at other issues to help us.
Thanks

**Pull request closed without merging**